src/main/java/baritone/behavior/PathingBehavior.java

```diff
@@ -565,4 +577,49 @@ private static AbstractNodeCostSearch createPathfinder(BlockPos start, Goal goal
     public void onRenderPass(RenderEvent event) {
         PathRenderer.render(event, this);
     }
+
+    @Override
+    public boolean isCloseToHome() {
+        return isCloseToHome;
+    }
+
+    @Override
+    public void updateIsCloseToHome() {
+        try {
+            IBaritone bar = BaritoneAPI.getProvider().getPrimaryBaritone();
+            EntityPlayerSP player = bar.getPlayerContext().player();
+            IWaypoint[] waypoints = ForWaypoints.getWaypointsByTag(bar, IWaypoint.Tag.HOME);
+            if (waypoints.length > 0) {
+                for (IWaypoint wp : waypoints) {
+                    boolean isClose = wp.getLocation().getDistance((int) Math.round(player.posX), (int) Math.round(player.posY), (int) Math.round(player.posZ)) <= Baritone.settings().homeProtectionRange.value;
+                    if (isClose) {
+//                    Helper.HELPER.logDirect("true");
```

Comment:
Remove debugging code in comments (ideally before committing)

Reply:
Changed in latest commit